src/main/resources/assets/tconstruct/lang/uk_ua.json

```diff
@@ -1445,7 +1445,7 @@
 
   "material.tconstruct.nahuatl": "Ацтекське дерево",
   "material.tconstruct.nahuatl.format": "%s з ацтекського дерева",
-  "material.tconstruct.nahuatl.flavor": "Це не обсидіанова деталь, якби такою була, то так би й написали: \«Обсидіан.\»",
+  "material.tconstruct.nahuatl.flavor": "Це не обсидіанова деталь, якби такою була, то так би й написали: \"Обсидіан.\"",
```

Comment:
the problem here is the `\`, not the non-english quote character. Whoever translated this did not realize that `\"` is how you type a quite character so thought they needed the `\` for their non-english quotes